server/games/game.py

```diff
@@ -242,6 +247,16 @@ def connections(self):
 
     @property
     def teams(self):
+        """
+        A dictionary of lists representing teams
```

Comment:
It returns a set not a dict. It would look like `{2, 3}` assuming that there are 2 teams

Reply:
Copypasted from BaseGame, didn't touch it yet.

Reply:
Can you please fix or delete this comment, because it's wrong